src/main/java/seedu/address/model/ingredient/IngredientName.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.model.ingredient;
+
+/**
+ * A class to represent the ingredientName in an ingredient
+ */
+
+public class IngredientName {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Ingredient's name should only contain alphabets and spaces, and it should not be blank.";
+
+    public static final String VALIDATION_REGEX_INGREDIENTNAME = "[a-zA-Z\\s]*";
+
+
+    // Identity fields
+    private final String ingredientName;
+
+    /**
+     * Constructs a {@code IngredientName}.
+     *
+     * @param name A valid ingredient name corresponding to VALIDATION_REGEX
+     */
+    public IngredientName(String name) {
+        this.ingredientName = name;
```

Comment:
should exercise some defensive programming here- what happens if someone calls your constructor without checking the regex? 

might want to throw exception (this applies to the other parts as well- i'm only writing this comment once).